webcompat/templates/contributors.html

```diff
@@ -2,80 +2,30 @@
-            <button class="contributors__item__btn is-active r-ResetButton" type="button">
-              How to Reproduce a Bug
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
-        </div>
-      </div>
-      <div class="contributors__item__content wc-UIContent is-open">{% include "contributors/reproduce-bug.html" %}</div>
-    </div>
-    <div class="contributors__item">
-      <div class="contributors__item__header">
-        <div class="wc-UIContent contributors__item__title">
-          <h2 id="diagnose">
-            <button class="contributors__item__btn r-ResetButton" type="button">
-              How to Diagnose a Bug
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
-        </div>
-      </div>
-      <div class="contributors__item__content wc-UIContent">{% include "contributors/diagnose-bug.html" %}</div>
-    </div>
-    <div class="contributors__item">
-      <div class="contributors__item__header">
-        <div class="wc-UIContent contributors__item__title">
-          <h2 id="outreach">
-            <button class="contributors__item__btn r-ResetButton" type="button">
-              How to Do Site Outreach
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
-        </div>
-      </div>
-      <div class="contributors__item__content wc-UIContent">{% include "contributors/site-outreach.html" %}</div>
-    </div>
-    <div class="contributors__item">
-      <div class="contributors__item__header">
-        <div class="wc-UIContent contributors__item__title">
-          <h2 id="contribute">
-            <button class="contributors__item__btn r-ResetButton" type="button">
-              Contribute to this site
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--disabled wc-Icon--minus" aria-hidden="true"></span>
-              <span class="wc-Icon wc-Icon--contributors wc-Icon--contributors--active wc-Icon--plus" aria-hidden="true"></span>
-            </button>
-          </h2>
+  {% include "shared/nav.html" %}
+  <main role="main">
+    <section class="grid grid-show-gap">
+      <div class="grid-row">
+        <aside class="grid-cell x1-fix">
+          <ol class="sub-nav" class="sub-nav">
+            <li class="sub-nav-header">Table of Contents</li>
```

Comment:
Do we really need "Table of Contents" here? It sounds a bit like the infamous "Click Here".

Reply:
This is content related and has his own bug: https://github.com/webcompat/webcompat.com/issues/1933